content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -165,56 +167,33 @@ When reviewing, use the following as a starting point.
 
 ### Blog
 
-- Early feedback on blog posts is welcome via a Google Doc or HackMD. Please request input early from the [#sig-docs-blog Slack channel](https://kubernetes.slack.com/archives/CJDHVD54J).
-- Before reviewing blog PRs, be familiar with [Submitting blog posts and case studies](/docs/contribute/new-content/blogs-case-studies/).
-- We are willing to mirror any blog article that was published to https://kubernetes.dev/blog/ (the contributor blog) provided that:
```

Comment:
I think we should add the approach that we've been informally using, namely that reviews should be done in the "original" repository to avoid duplicate (and potentially conflicting) reviews. Does this make sense?

Reply:
Yes with a caveat - ideally those happen at HackMD / Google Docs stage.
Would you be willing to write a suggestion on the change to make?

Reply:
I am, will submit one shortly.